server_addon/houdini/client/ayon_houdini/api/hda_utils.py

```diff
@@ -418,3 +421,74 @@ def keep_background_images_linked(node, old_name):
 
     if changes:
         set_background_images(parent, images)
+
+
+def get_products_menu_items(node):
+    """Return products menu items
+
+    It gets a list of available products of the specified product types
+      within the specified folder path with in the specified project.
+
+    Users can specify those in the HDA parameters.
+
+    Returns:
+        List[str]: Product options for Products menu.
+    """
+    project_name = node.evalParm("project_name")
+    folder_path = node.evalParm("folder_path")
+    representation_name = node.evalParm("representation_name")
+
+    folder_entity = ayon_api.get_folder_by_path(project_name,
+                                                folder_path,
+                                                fields={"id"})
+    if not folder_entity:
+        return []
+
+    products = ayon_api.get_products(
+        project_name,
+        folder_ids=[folder_entity["id"]],
+        product_types=[representation_name]
+    )
+
+    items = []
+    for product in products:
+        items.append(product["name"])
+        items.append(product["name"])
+
+    return(items)
+
+
+def select_folder_path(node):
+    """select folder path.
+
+    When triggered it opens a dialog shows the available
+      folder paths within a given project.
```

Comment:
```suggestion
    When triggered it opens a dialog that shows the available
    folder paths within a given project.
```

Reply:
solved in https://github.com/BigRoy/ayon-core/pull/9/commits/73a9833db2ac9d3621362995812788ae5d5ec46f